File: Java/Problem1.java
public class Problem1 {

	public static void main(String v[]) {
		double start = System.nanoTime();
		int num = 0;

		System.out.println("Find Multiples of 3 and 5");

		for (int i = 3; i < 1000; i++) {
			if (i % 3 == 0) {
				num += i;
			}
			if (i % 5 == 0) {
				num += i;
			}
		}
		System.out.println(num);
		double end = System.nanoTime();
		System.out.println("Time Elapsed is: " + String.valueOf((end - start))
				+ "ns");
	}
}

File: Python/Problem1.py
'''
Created on Aug 16, 2014

@author: Prathyush
'''
<<<<<<< HEAD
import time
print "To find the multiples of 3 and 5"

start = time.time()
=======
from main import test
print "To find the multiples of 3 and 5"

start = test.Time()
>>>>>>> 71df3e73de1f7d326f1eeda65a9ba5fef9b87c38
num=0

for i in xrange (3,1000):
    if i%3==0:
        num+=i
    elif i%5==0:
        num+=i;
    

print num
<<<<<<< HEAD
end = time.time()
=======
end = test.Time()
>>>>>>> 71df3e73de1f7d326f1eeda65a9ba5fef9b87c38
print 'Elapsed test: '+str(end-start)+'s'

